src/steiner_tree.jl

```diff
@@ -9,5 +9,13 @@ using SimpleTraits: SimpleTraits, Not, @traitfn
     map(v -> vertex_positions(g)[v], term_vert),
     dist_matrix_to_position_dist_matrix(g, distmx),
   )
-  return typeof(g)(position_tree, map(v -> ordered_vertices(g)[v], vertices(position_tree)))
+  named_st = typeof(g)(
```

Comment:
```suggestion
  tree = typeof(g)(
```
I've been using the convention that I call graphs `graph` or `g`, trees `tree` or `t`, etc. in this package, with the presumption that they are all named or generic and then underlying simple/integer vertex graphs are `position_graph`, `position_tree`, etc. (meaning they are graphs where the vertices are the integer positions of the vertices of the corresponding named graphs in the ordered list of named vertices).

Reply:
Makes sense – I'll change that